core/runtime/src/main/java/io/quarkus/runtime/ApplicationLifecycleManager.java

```diff
@@ -135,7 +137,11 @@ public static void run(Application application, Class<? extends QuarkusApplicati
                 }
             }
         } catch (Exception e) {
-            Logger.getLogger(Application.class).error("Error running Quarkus application", e);
+            if (appStarted) {
+                //we only log if the error occurred after the application was started
+                //as the generated application class already has logging
+                Logger.getLogger(Application.class).error("Error running Quarkus application", e);
```

Comment:
So what happens if the application did not start? We don't have any error message?

Reply:
The generated Application class has its own logging.